settings.gradle

```diff
@@ -256,6 +256,10 @@ include(':open-metadata-implementation:view-services:dino-view:dino-view-api')
 include(':open-metadata-implementation:view-services:dino-view:dino-view-server')
 include(':open-metadata-implementation:view-services:dino-view:dino-view-spring')
 include(':open-metadata-implementation:view-services:dino-view')
+include(':open-metadata-implementation:view-services:server-author-view:server-author-view-api')
+include(':open-metadata-implementation:view-services:server-author-view:server-author-view-server')
+include(':open-metadata-implementation:view-services:server-author-view:server-author-view-spring')
+include(':open-metadata-implementation:view-services:server-author-view')
```

Comment:
Great so far, but we also need project definitions ie similar to these we use for dino:
```
project(':open-metadata-implementation:view-services:dino-view:dino-view-server').projectDir = file('open-metadata-implementation/     view-services/dino-view/dino-view-server')
```
You'll need to also add those to settings.gradle for the above projects.

We shouldn't need the top level directory if empty (server-author-view) but we still have them for other projects, so ok to leave for consistency (I'll clean this area up later when I get back to more gradle work)

Reply:
ok done